static/src/javascripts/projects/commercial/modules/close-disabled-slots.js

```diff
@@ -1,9 +1,25 @@
 // @flow
 import qwery from 'qwery';
 import fastdom from 'lib/fastdom-promise';
+import once from 'lodash/functions/once';
+import { dfpEnv } from 'commercial/modules/dfp/dfp-env';
 import { commercialFeatures } from 'commercial/modules/commercial-features';
 
-const adSlotSelector: string = '.js-ad-slot';
+const shouldDisableAdSlot = adSlot =>
+    window.getComputedStyle(adSlot).display === 'none';
+
+const closeDisabledSlots = once((): Promise<void> => {
+    // Get all ad slots
+    let adSlots: Array<Element> = qwery(dfpEnv.adSlotSelector);
```

Comment:
I don't know if that's true. Although that list would be a list of things considered a safe-read, it doesn't necessarily mean all other reads should be outside of fastdom. Think of this specific example. If the code is written to perform fastdom-write a node removal, immediately prior to the query selector, then the `qwery` would need to be in a fastdom-read, despite being safe from reflow.